x/mobileproxy/mobileproxy.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package mobileproxy provides convenience utilities to help applications run a local proxy
+// and use that to configure their networking libraries.
+//
+// This package is suitable for use with Go Mobile, making it a convenient way to integrate with mobile apps.
+package mobileproxy
```

Comment:
This should also work for desktops as well, right? Maybe rename to `localwebproxy`?

Reply:
This library is intended for Go Mobile only.
It won't work for generating a C library, for instance.